### PR TITLE
Feature/homepage - Project initial setup and homepage UI

### PR DESCRIPTION
**Pull request by vidhya-metacell**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (tarelli) [inline LICENSE:1]: Where does this come from?

File: _posts/2014-07-13-project-6.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (tarelli) [inline _posts/2014-07-13-project-6.markdown:1]: Are these used?

File: css/font-awesome/css/graphik.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
+@font-face {
```
Comment (tarelli) [inline css/font-awesome/css/graphik.css:1]: Are we using Graphik in the end?

**Review by tarelli — CHANGES_REQUESTED** [review]
Thanks @vidhya-metacell great job on the fidelity! I left a few minor comments, even if it's the first commit can we go through all files and make sure we remove everything that's not needed/used? I have also seen there are minified and not minified libraries, are the latter needed?

File: LICENSE (hunk 1 above)
Comment (vidhya-metacell) [reply LICENSE:1]: This licence came with the theme, so I am not sure if we want to keep or delete it

File: css/font-awesome/css/graphik.css (hunk 3 above)
Comment (vidhya-metacell) [reply css/font-awesome/css/graphik.css:1]: No, we will use Inter. This was made with Graphik , it was before we decided to change the font. I'll update

**Comment by vidhya-metacell** [comment]
> Thanks @vidhya-metacell great job on the fidelity! I left a few minor comments, even if it's the first commit can we go through all files and make sure we remove everything that's not needed/used? I have also seen there are minified and not minified libraries, are the latter needed?

@tarelli I want to keep the js libraries cause first commit I am not sure which one we would need. Will do the clean up for the libraries, once done with all the pages. Hope that's ok.

**Review by tarelli — APPROVED** [review]

**Pull request merged**